### PR TITLE
add flowId to logs

### PR DESCRIPTION
**Pull request by abobr**
https://www.jetbrains.com/help/teamcity/build-script-interaction-with-teamcity.html#BuildScriptInteractionwithTeamCity-ReportingTests

https://www.jetbrains.com/help/teamcity/build-script-interaction-with-teamcity.html#Message-FlowId

When running tests inside a container (docker 19.03) on TeamCity agent (VM - ubuntu 18.04)
teamcity treats tests as failed. adding flowId helps to solve this issue.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -5,10 +5,11 @@
 'use strict';
 
 const formatter = require('./formatter');
+const flowId = process.env['JEST_TEAMCITY_FLOWID'] || process.pid.toString();
```
Comment (itereshchenkov) [inline lib/index.js]: It looks like it's not jest specific feature. Could you rename it to `TEAMCITY_FLOWID`?

Comment (abobr) [reply lib/index.js]: Done

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jest-teamcity",
-  "version": "1.4.0",
+  "version": "2.0.0",
```
Comment (itereshchenkov) [inline package.json]: and could I ask you to revert the version as well? I will dump the version when it's released

Comment (abobr) [reply package.json]: ok

**Comment by itereshchenkov** [comment]
@abobr thanks a lot for your contribution. I will publish a new version today

**Comment by abobr** [comment]
@itereshchenkov hi ) any news on when it could be merged?

**Comment by itereshchenkov** [comment]
Merged and published. Thanks for your support!

**Comment by abobr** [comment]
Thank you.

**Pull request merged**